src/wincng.c

```diff
@@ -390,15 +488,45 @@ _libssh2_wincng_init(void)
     }
 
     ret = BCryptOpenAlgorithmProvider(&_libssh2_wincng.hAlgDH,
-                                      BCRYPT_DH_ALGORITHM, NULL, 0);
+                                      BCRYPT_DH_ALGORITHM, NULL,
+                                      0);
```

Comment:
```suggestion
                                      BCRYPT_DH_ALGORITHM, NULL, 0);
```
Small nit, we can leave formatting as-is.